HW0/P6/P6B.py

```diff
@@ -16,19 +18,31 @@ def burnTime(t):
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+
+    # Generate 16 random wait times
+    wait_time = np.logspace(-6, 0, 25)#[random.uniform(10**(-6), 1) for x in range(100)]
 
     for t in wait_time:
+        print 'On wait time', t, '!'
         # Compute jobs serially and in parallel
         # Use time.time() to compute the elapsed time for each
-        serialTime = 1
-        parallelTime = 1
+
+        # Calculate the time it take serially
+        serialTime = time.time()
+        for ii in range(N):
```

Comment:
You should probably use `xrange` here, which returns a generator instead of a list, as it is an advantage of the serial program that it does not need to save a list of numbers to memory, it just needs each number once.